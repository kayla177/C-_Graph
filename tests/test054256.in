ENTITY V2CZW9DZ name_V2CZ institution
ENTITY HRMSE1OR name_HRMS conference
ENTITY 57686Y71 name_5768 journal
RELATIONSHIP V2CZW9DZ cited_by HRMSE1OR 8.0
PRINT V2CZW9DZ
HIGHEST
EXIT
