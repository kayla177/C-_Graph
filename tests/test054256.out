success
success
success
success
HRMSE1OR
V2CZW9DZ HRMSE1OR 8
